src/starknetCli.ts

```diff
@@ -210,18 +225,26 @@ export async function runStarknetCallOrInvoke(
     }
     throw e;
   }
-
   try {
-    execSync(
+    let warpOutput: string = execSync(
       `${warpVenvPrefix} starknet ${callOrInvoke}  --address ${options.address} --abi ${abiPath} --function ${funcName} --network ${options.network} ${wallet} ${account} ${inputs}`,
-      { stdio: 'inherit' },
-    );
+    ).toString('utf-8');
+
+    if (isCall && !options.use_cairo_abi) {
+      const decodedOutputs = await decodeOutputs(
+        solAbiPath ?? '',
+        options.function,
+        warpOutput.toString().split(' '),
```

Comment:
Now that we have the components of a struct in the abi we can improve the formatting and put the key names for a struct and for arrays we should wrap them in []